static/js/controllers/reports.js

```diff
@@ -250,17 +335,18 @@ var _ = require('underscore'),
             _query();
           }
         }
-
       };
 
       $scope.$on('ClearSelected', function() {
-        $scope.selected = null;
+        $scope.selected = [];
+        $('#reports-list input[type="checkbox"]')
+          .prop('checked', false);
         liveList.clearSelected();
```

Comment:
Additional point : that piece of checkboxing is alone in livelist: 
https://github.com/medic/medic-webapp/blob/cfae7ff81efb4114a1ac222a667080d83e88f158/static/js/services/live-list.js#L96

You could pull it out of livelist and into the wrapper, and thus do all the checkboxing in the same place.